projects/storefrontapp/tsconfig.app.prod.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "paths": {
       "@spartacus/setup": ["dist/setup"],
+      "@spartacus/setup/prerender": ["dist/setup/prerender"],
       "@spartacus/setup/ssr": ["dist/setup/ssr"],
```

Comment:
Let's discuss the structure. What about:
`setup/srr`
`setup/ssr/prerender`
`setup/ssr/app`
?

Why I ask:
In https://github.com/SAP/spartacus/pull/16361/files I plan to introduce a `BaseServerModule` to be imported in client's `AppServerModule`. My new module would be responsible for any core functionalities that are needed for SSR. For example it would provide BEFORE_APP_SERIALIZED hook that transfers errors from the App to the OptimizedSsrEngine. I guess I will need to put this Angular code in a different entry point than `/setup/ssr`, becasue it already contains native Node code.

Reply:
As discussed offline, we decide to leave it as is.
The import for customers will `@spartacus/setup/prerendering`